app/index/activities/controller.js

```diff
@@ -19,10 +20,13 @@ export default Controller.extend({
    *
    * @property {Activity[]} _allActivities
    * @private
-   */
-  @computed()
-  _allActivities() {
+  _allActivities: computed(function() {
     return this.store.peekAll('activity')
+  }),
+   */
+  init() {
+    this._super(...arguments)
+    this.set('_allActivities', this.store.peekAll('activity'))
```

Comment:
Why are you doing it that way?

Reply:
Instead of having an empty computed, which also only runs once when it gets called the first time.
I put it into init for the same effect.